Plugin/Energistics/Mapping/WitsmlWellboreCompletionPerforationToVtkPolyData.cxx

```diff
@@ -16,7 +16,7 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -----------------------------------------------------------------------*/
-#include "WitsmlWellboreCompletionPerforationToVtkPolydata.h"
+#include "Mapping/WitsmlWellboreCompletionPerforationToVtkPolyData.h"
 
```

Comment:
Build fails on VS2017....

I need a "#include <array>" here

Reply:
I'll do it